### PR TITLE
c_sharp_basics_1

### PR DESCRIPTION
**Pull request by 9inebar**

### CODE REVIEW

File: .idea/.idea.Aleh_Hramovich_Application/.idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Default ignored files
```
Comment (MariaEroshevich) [inline .idea/.idea.Aleh_Hramovich_Application/.idea/.gitignore:1]: move gitignore to folder with solution.

Comment (MariaEroshevich) [reply .idea/.idea.Aleh_Hramovich_Application/.idea/.gitignore:1]: and .idea files also shouldn't be pushed to remote repository.

**Review by MariaEroshevich — CHANGES_REQUESTED** [review]

File: .idea/.idea.Aleh_Hramovich_Application/.idea/.gitignore (hunk 1 above)
Comment (9inebar) [reply .idea/.idea.Aleh_Hramovich_Application/.idea/.gitignore:1]: @MariaEroshevich looks like it's already in the folder contacting sln file:
Screenshot 2023-06-02 at 21 58 09

File: .gitignore [hunk 2]
```diff
@@ -2,94 +2,4 @@ bin/
...
-fastlane/test_output
-
-# Code Injection
-#
-# After new code Injection tools there's a generated folder /iOSInjectionProject
-# https://github.com/johnno1962/injectionforxcode
-
-iOSInjectionProject/
\ No newline at end of file
+.idea
```
Comment (MariaEroshevich) [inline .gitignore]: copy git ignore from the link https://gist.github.com/kmorcinek/2710267 and paste to your gitignore file.

**Review by MariaEroshevich — CHANGES_REQUESTED** [review]

**Review by MariaEroshevich — APPROVED** [review]

**Pull request merged**